docs/extensions/unreal_engine.md

```diff
@@ -0,0 +1,34 @@
+# Exporting your model to NVOL for Unreal Engine
+
+<p align="center">
+    <img width="800" alt="image" src="./imgs/desolation_unreal.png">
+</p>
+
+
+## Overview
+
+NVOL is a new standard file format to store NeRFs in a fast and efficient way. NVOL files can be obtained from NeRFStudio checkpoints files (.ckpt) using Volinga Suite (https://volinga.ai/).
```

Comment:
`NVOL is a file format to store NeRFs in a fast and efficient way. NVOL files can be obtained from NeRFStudio checkpoints files (.ckpt) using the [Volinga Suite](https://volinga.ai/)`